1zadanie.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="1zadanie.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:200,300,300i,400,400i,700,700i&amp;subset=latin-ext" rel="stylesheet">
+    
+    </head>
+<body>
+
+    
+     <section id="layout" >
+         <img src="coming-soon-ribbon.png" id="coming_soon">
+         <img src="undraw_developer_activity_bv83.svg" id="guy" >
+         <div id="code" style="color: white;">CODE ACADEMY</div>
```

Comment:
1. Nie stosuj styli inline. To nie jest dobre.
2. Code Academy. Plik HTML ma być pisany bez najmniejszych oznak "pokemona". Zamiast wielkich liter użyj w css
```css
text-transform: uppercase;
```